test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -779,6 +783,58 @@ func (so sleepOp) patchParams(_ *workload) (realOp, error) {
 	return &so, nil
 }
 
+// startCollectingMetricsOp defines an op that starts metrics collectors.
+// stopCollectingMetricsOp has to be used after this op to finish collecting.
+type startCollectingMetricsOp struct {
+	// Must be "startCollectingMetrics".
+	Opcode operationCode
+	// Name appended to workload's name.
+	Name string
```

Comment:
what is it for?

Reply:
For default metrics collection within createPodsOp we append namespace name to the name of workload in the results json. In this case I added this argument to use some user-defined name if we can't use the namespace directly. Especially we need to remember that collecting the metrics multiple time within one workload is not forbidden as long as they don't interleave.